lib/puppet_litmus/rake_tasks.rb

```diff
@@ -83,6 +83,28 @@ def run_local_command(command)
     end
   end
 
+  desc "provision list of machines from provision.yaml file. 'bundle exec rake 'litmus:provision_list[default]'"
+  task :provision_list, [:key] do |_task, args|
+    provision_hash = YAML.load_file('./provision.yaml')
+    provisioner = provision_hash['default']['provisioner']
+    provision_hash[args[:key]]['images'].each do |image|
+      include PuppetLitmus
+      Rake::Task['spec_prep'].invoke
+      config_data = { 'modulepath' => File.join(Dir.pwd, 'spec', 'fixtures', 'modules') }
+      raise "the provision module was not found in #{config_data['modulepath']}, please amend the .fixtures.yml file" unless File.directory?(File.join(config_data['modulepath'], 'provision'))
```

Comment:
This is repeated in various tasks. It'd be good to at least add a helper for this.